scripts/registration.js

```diff
@@ -0,0 +1,32 @@
+import { REGISTER_API_URL } from "./constants.js";
+import { fetchData } from "./fetchData.js";
+
+const form = document.querySelector('#registration-form');
+const name = document.querySelector('#reg-name');
+const email = document.querySelector('#reg-email');
+const password = document.querySelector('#reg-password');
+
+async function registerUser(user) {
+    const postBody = JSON.stringify(user);
```

Comment:
You need a try catch here like in the loginUser function. Display a message in the html if the call is a success or a failure. See the loginUser comment.